samples/dotnet/kernel-syntax-examples/Resources/EmbeddedResource.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.IO;
+using System.Reflection;
+using RepoUtils;
+
+namespace Resources;
+
+internal static class EmbeddedResource
```

Comment:
This class does some funky stuff and could totally use a comment or two and how and and why it does.

Reply:
sure I can add comment, but it's nothing special (or funky) :-) It's just loading resources from the assembly.